src/Semantic/CLI.hs

```diff
@@ -114,7 +114,9 @@ parseCommand = command "parse" (info parseArgumentsParser (progDesc "Generate pa
                       <$> option str (long "gitDir" <> help "A .git directory to read from")
                       <*> option shaReader (long "sha" <> help "The commit SHA1 to read from")
                       <*> ( ExcludePaths <$> many (option str (long "exclude" <> short 'x' <> help "Paths to exclude"))
-                        <|> ExcludeFromHandle <$> flag' stdin (long "exclude-stdin" <> help "Exclude paths given to stdin"))
+                        <|> ExcludeFromHandle <$> flag' stdin (long "exclude-stdin" <> help "Exclude paths given to stdin")
+                        <|> OnlyPaths <$> many (option str (long "only" <> help "Only include the specified paths"))
+                        <|> OnlyPathsFromHandle <$> flag' stdin (long "only-stdin" <> help "Include only the paths given to stdin"))
```

Comment:
The symmetry centre of my brain suggests: perhaps `IncludePaths` and `IncludeFromHandle` or `IncludePathsFromHandle` to mirror the exclude ones? I’m not super fussed, tho.

Reply:
I like that. I'm going to leave the command line switches because I think `--only` is a better description of the behavior, but I can rename the type constructors.